recipes-multimedia/libcamera-apps/libcamera-apps.bb

```diff
@@ -0,0 +1,35 @@
+SUMMARY = "Raspberry libcamera-apps"
+
+LICENSE = "BSD-2-Clause"
+LIC_FILES_CHKSUM = "file://license.txt;md5=a0013d1b383d72ba4bdc5b750e7d1d77"
+
+SRC_URI = "git://github.com/raspberrypi/libcamera-apps.git;branch=main;protocol=https"
+
+PV = "1.0+git${SRCPV}"
```

Comment:
There is no such version as far as I can see. I don't see any 1.0 tags in the repo. Just use `git` in the recipe filename.